docs/machineregistration-reference.md

```diff
@@ -104,6 +104,45 @@ The only required value for a successful installation is the `device` key as we
   ```
 </details>
 
+#### config.elemental.reset
+
+Contains the reset configuration that would be applied via `elemental-register --reset`, when booted from the recovery partition and passed to [`elemental reset`](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental_reset.md)
+
+Supports the following values:
+
+| Key         | Type   | Default value | Description                                                                                                                                |
+|-------------|--------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| enabled           | bool   | false  | Allow reset on all registered machines  |
+| reset-persistent  | bool   | true   | Format the COS_PERSISTENT partition     |
+| reset-oem         | bool   | true   | Format the COS_OEM partition            |
+| config-urls       | list   | empty  | Cloud-init config files                 |
+| system-uri        | string | empty  | Sets the system image source and its type (e.g. 'docker:registry.org/image:tag') instead of using the running ISO                          |
+| debug             | bool   | false  | Enable debug output                     |
+| poweroff          | bool   | false  | Shutdown the system after reset         |
+| reboot            | bool   | true   | Reboot the system after reset           |
+
```

Comment:
What happens if "poweroff" and "reset" are both "true" ?
What happens if both are "false" ?

Not sure if having two boolean settings is ideal here ? 🤔

Reply:
This is equivalent to the `install` configuration.  
As far I understand we don't have any validation on the object side, so you are able to configure both true.  

If that's the case, the `reboot` should "win", the machine will reboot.

If you configure both false, the machine will reset and stay in recovery mode. This can be useful for debugging.

Reply:
This should be clearly documented. (Esp. the debugging possibility 😉 )

Reply:
Added an info note to clarify that.